src/popup.html

```diff
@@ -158,6 +158,18 @@ <h4 class="moz-vpn-logotype vpn-status-container-list display-none">Mozilla VPN
       </h4>
     </div>
     <div class="scrollable identities-list">
+      <div class="searchbar">
+        <label for="search-terms"
+               class="hide-label"
+               data-i18n-message-id="filterInputLabel">
```

Comment:
I don't see this (`Filter container list by name:`) in the GIF. How is this used?

Reply:
The label is for screen reader only.